doc/python/shapes.md

```diff
@@ -803,4 +862,32 @@ fig.show(config={'modeBarButtonsToAdd':['drawline',
 ```
 
 ### Reference
-See https://plotly.com/python/reference/layout/shapes/ for more information and chart attribute options!
\ No newline at end of file
+See https://plotly.com/python/reference/layout/shapes/ for more information and chart attribute options!
+
+<!-- #region -->
```

Comment:
this "what about Dash" section gets added automatically at build time, and so should not be included here please